Samples/Swift/DaysUntilBirthday/Shared/Services/VerificationLoader.swift

```diff
@@ -53,6 +116,37 @@ final class VerificationLoader: ObservableObject {
       self.verifiedAgeViewModel.verificationState = .verified(verifyResult)
     }
   }
+
+  private var cancellable: AnyCancellable?
+
+  /// Fetches the age verification signal of the current user.
+  func fetchAgeVerificationSignal(verifyResult: GIDVerifiedAccountDetailResult,
+                                  completion: @escaping () -> Void) {
+    self.verificationPublisher(verifyResult: verifyResult) { publisher in
+      self.cancellable = publisher.sink { sinkCompletion in
+        switch sinkCompletion {
+        case .finished:
+          break
```

Comment:
Is a `break` technically needed?

Reply:
I believe so. Otherwise, I receive error `'case' label in a 'switch' must have at least one executable statement`. I believe a `break` or `()` is needed to break out of the switch and indicate you intend for the case to do nothing.

Reply:
Ohh I see what you mean. I've switched this over to an if statement so it only focuses on the error case and confirmed it works.